plothist/comparison.py

```diff
@@ -40,9 +40,11 @@ def _is_unweighted(hist):
     return np.allclose(hist.variances(), hist.values())
 
 
-def get_poisson_uncertainties(hist):
+def get_asymmetrical_uncertainties(hist):
     """
-    Get Poisson asymmetrical uncertainties for a histogram.
+    Get Poisson asymmetrical uncertainties for a histogram via a frequentist approach based on a confidence-interval computation.
```

Comment:
Should we also put the link to the uncertainty presentation? Or maybe this will be in the doc.

Reply:
It will be in the doc (another PR).